playbooks/roles/shadowsocks/defaults/main.yml

```diff
@@ -4,3 +4,6 @@ shadowsocks_local_port: "1080"
 shadowsocks_timeout: "600"
 shadowsocks_encryption_method: "chacha20-ietf-poly1305"
 shadowsocks_tcp_fast_open: "true"
+shadowsocks_obfs_plugin: "obfs-server"
+shadowsocks_obfs_plugin_opts: "obfs=http"
+shadowsocks_obfs_client_plugin_opts: "obfs=http;obfs-host=www.xxx.com"
```

Comment:
I'll change it to `www.cloudfront.net`. Didn't know that there is a website with this host name which is highly NSFW.